examples/ocis-compose/config/identifier-registration.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
how is this file used in the composer file?

Reply:
it is used in `KONNECTD_IDENTIFIER_REGISTRATION_CONF: /etc/ocis/identifier-registration.yml` in the environment.

Reply:
I see, and `config` is used by the volumes